src/app/(dashboard)/a/settings/component/setting-card.tsx

```diff
@@ -0,0 +1,112 @@
+'use client'
+import { Button } from "@/components/ui/button"
+import { Card, CardHeader, CardTitle } from "@/components/ui/card"
+import { Label } from "@/components/ui/label"
+import Link from "next/link"
+import { useForm } from "react-hook-form"
+
+
+export default function SettingsCard() {
+  const {
+    register,
+    handleSubmit
+  } = useForm({})
+
+  const onSubmit = async (formData: any) => {
+    try {
+      formData.port = parseInt(formData.port, 10);
+      formData.secure = parseBoolean(formData.secure);
+      const response = await fetch('/api/settings', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(formData),
+      });
+
+      if (response.ok) {
+        console.log('Nice!!!!!!!!!!');
```

Comment:
Replace all console.log() or replace with Toast UI if you want to display a useful message after an action.